### ISSUE TITLE
When the browser window is resized the height of the graph is reduced.

### ISSUE THREAD

**Issue by dazhbog** [masayuki0812/c3#877]
Hi there,

Thanks for your work on this awesome library.

I've been having this issue lately where when my browser window is very wide, the graph doesnt seem to respect the height of its parent, the height set in HTML(inline CSS) or the size parameter set in javascript by the size:{height:400}.

The effect is shown below. 

![screenshot 2015-01-09 05 36 04](https://cloud.githubusercontent.com/assets/1836262/5677190/78251acc-97c1-11e4-98ec-a2fe48279444.png)
Displays properly here at a width of 1300px

![screenshot 2015-01-09 05 35 43](https://cloud.githubusercontent.com/assets/1836262/5677189/7823a3ae-97c1-11e4-880e-dc79c9074bf6.png)
Above, the width is around 1920px and the graph's height is reduced

Any ideas? Thanks. (tried on 0.4.8 & 0.4.4)

**Comment by hansen90** [comment]
I had the same issue. I fixed it by adding a min-height style to the div element. For example,

Before: <div id="chart"></div>

After: <div id="areaChart" style="min-height: 400px"></div>

**Comment by hansen90** [comment]
I had the same issue. I fixed it by adding a min-height style to the div element. For example,
```
Before: <div id="chart"></div>
```
```
After: <div id="chart" style="min-height: 400px"></div>
```

**Comment by dazhbog** [comment]
Hi there,

I tried that but the graph still doesn't seem to get the full height.

**JS Fiddle showing the issue**
http://jsfiddle.net/dazhbog/1rb8qexp/

**Comment by masayuki0812** [comment]
Hi, sorry I can't see this issue. It's still happening in the fiddle?

**Comment by dazhbog** [comment]
This is very interesting. I just went to the fiddle and it seems that its working now without any change. I then went to my local environment and I could still see the problem, I just made sure I re-update the library to the same version as the masayuki0812/c3/master/c3.js (latest at the moment 4.8) and it seems to be fine now.

I shall close it for now then, thanks for your time! :+1:

**Comment by dazhbog** [comment]
Ok this is weird, Is there a difference between this file here https://rawgit.com/masayuki0812/c3/master/c3.js

and the file in the .js file in the zip file here?
https://github.com/masayuki0812/c3/archive/0.4.8.zip

Both have this line     var c3 = { version: "0.4.8" };

I had the one from the zip file, where the bug is present but when I use the one from this link https://rawgit.com/masayuki0812/c3/master/c3.js everything works. 

I think some new changes made the fiddle to start working after it was created.

**Comment by masayuki0812** [comment]
Yes, there is. https://rawgit.com/masayuki0812/c3/master/c3.js is different from the latest release (v0.4.8 in this case) and includes the fixes that are not released yet. It might be better to create develop branch for the next version and work on it.. Anyway, sorry for the confusion.

**Comment by dazhbog** [comment]
Ok I have updated the fiddle to use the 0.4.8 release so that you can quickly check the issue out.
same link: http://jsfiddle.net/dazhbog/1rb8qexp/

I will go ahead and open the issue again, If you think its fixed in the 0.4.9 release the you can go ahead and close it again.

I also tried the 0.4.9, that seems to fix the issue, but had another issue that the graphs wouldnt appear until I resized the window. But i guess I have to wait until it is a proper release to test it.

Thanks.

**Comment by masayuki0812** [comment]
Please let me close this issue. For another issue you found, can you open a new issue? Thanks.
